tests/common_test.go

```diff
@@ -97,6 +99,69 @@ func GetToolsConfig(sourceConfig map[string]any, toolKind, param_tool_statement,
 	return toolsFile
 }
 
+// GetHTTPToolsConfig returns a mock HTTP tool's config file
+func GetHTTPToolsConfig(sourceConfig map[string]any, toolKind string) map[string]any {
```

Comment:
```suggestion
func GetHTTPToolsConfig(sourceConfig map[string]any) map[string]any {
```

we probably don't need the toolKind here and can just hard code the "http" since this tools config is specifically just for the http tool.

Reply:
The current HTTP Tool only supports JSON content type (I have updated the tool kind to http-json for clarity), We may add in more tools (e.g http-xml, http-text) in the future.